lib/cubit/ln_address/ln_address_cubit.dart

```diff
@@ -159,6 +161,23 @@ class LnAddressCubit extends Cubit<LnAddressState> {
     emit(state.clearUpdateStatus());
   }
 
+  Future<String?> _getBolt12Offer() async {
+    try {
+      final PrepareReceiveRequest prepareRequest =
+          const PrepareReceiveRequest(paymentMethod: PaymentMethod.bolt12Offer);
+      final PrepareReceiveResponse? prepareResponse =
+          await breezSdkLiquid.instance?.prepareReceivePayment(req: prepareRequest);
+      if (prepareResponse != null) {
+        final ReceivePaymentRequest req = ReceivePaymentRequest(prepareResponse: prepareResponse);
+        final ReceivePaymentResponse? res = await breezSdkLiquid.instance?.receivePayment(req: req);
+        return res?.destination;
+      }
+    } catch (e) {
+      _logger.warning('Failed to get BOLT12 Offer', e);
+    }
+    return null;
+  }
```

Comment:
Syntactic suggestion for readability. We can forgo null-handling as they'll be caught by the catch block.
```suggestion
  Future<String?> _getBolt12Offer() async {
    try {
      final BindingLiquidSdk breezSdkLiquid = _breezSdkLiquid.instance!;

      const PrepareReceiveRequest prepareReq = PrepareReceiveRequest(
        paymentMethod: PaymentMethod.bolt12Offer,
      );
      final PrepareReceiveResponse prepareRes = await breezSdkLiquid.prepareReceivePayment(req: prepareReq);

      final ReceivePaymentRequest receiveReq = ReceivePaymentRequest(prepareResponse: prepareRes);
      final ReceivePaymentResponse receiveRes = await breezSdkLiquid.receivePayment(req: receiveReq);

      return receiveRes.destination;
    } catch (e, stackTrace) {
      _logger.warning('Failed to get BOLT12 Offer', e, stackTrace);
      return null;
    }
  }
```


May need to call `dart format -l 110 .` if committed through GitHub UI.